src/components/pickers/RangeSlider/styled.ts

```diff
@@ -0,0 +1,112 @@
+import { tasty } from '../../../tasty';
+
+export const StyledThumb = tasty({
+  styles: {
+    top: '2x',
+    width: '2x',
+    height: '2x',
+    fill: {
+      '': '#purple',
+      hovered: '#purple-text',
+      dragged: '#purple',
+      disabled: '#dark-04',
+    },
+    cursor: {
+      hovered: 'pointer',
+      dragged: 'grab',
```

Comment:
You either use the grab/grabbing combination or use just the pointer value only.